src/gerbil/gxi

```diff
@@ -1,11 +1,23 @@
 #!/bin/sh
 set -eu
 
+
 if [ -z "${GERBIL_HOME:-}" ]; then
-    GERBIL_HOME=$(dirname $(cd ${0%/*} && echo $PWD))
+    SCRIPT=$(readlink --canonicalize "$0")
+    GERBIL_HOME=$(dirname "${SCRIPT%/*}")
     export GERBIL_HOME
 fi
 
+
+if [ $# -gt 0 ]; then
+    case $1 in
+        --home)
```

Comment:
Same: on Nix, this would probably echo a fixed path.

Reply:
> But... can we somehow move all this logic to the scheme side?

I thought about that, but gxi require it before loading any gerbil files, and this is just easier, and likely faster, due to ports and strings and blah blah blah.